dashboard/app/models/census/census_summary.rb

```diff
@@ -28,4 +28,161 @@ class Census::CensusSummary < ApplicationRecord
   enum teaches_cs: TEACHES
 
   validates_presence_of :audit_data
+
+  def self.submission_teaches_cs(submission, is_high_school)
+    if is_high_school
```

Comment:
And this

Reply:
is_high_school is an argument to the method, not a method itself. I didn't think that variable names typically ended with a question mark.

Reply:
No they don't, and actually can't. If it's a variable, don't worry about it. I thought it was another method call.